src/features/apply-form/components/box/ProgressBox.tsx

```diff
@@ -0,0 +1,15 @@
+import { Progress } from '@/shared';
+
+type Props = {
+  step: number;
+  progressValue: number;
+};
+
+export const ProgressBox = ({ step, progressValue }: Props) => {
+  return (
+    <div className='flex w-2/5 items-center gap-3 py-3'>
+      <Progress value={progressValue} className='flex-1' />
+      <p className='text-sm font-semibold'>{step} / 2 페이지</p>
```

Comment:
```suggestion
export const ProgressBox = ({ page, progressValue }: Props) => {
  return (
    <div className='flex w-2/5 items-center gap-3 py-3'>
      <Progress value={progressValue} className='flex-1' />
      <p className='text-sm font-semibold'>{page} / 2 페이지</p>
```
사소한 거지만 한번 고쳐볼게요  
step이든 page든 하나로 통일하삼요..